src/main/java/com/ernie/TicketApp/repository/TicketDAO.java

```diff
@@ -0,0 +1,71 @@
+package com.ernie.TicketApp.repository;
+
+import com.ernie.TicketApp.model.Ticket;
+import com.ernie.TicketApp.model.TicketType;
+import com.ernie.TicketApp.model.User;
+
+import java.sql.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class TicketDAO {
+    static final String DB_URL = "jdbc:postgresql://localhost:5432/my_ticket_service";
+    static final String USER = "postgres";
+    static final String PASSWORD = "ernie";
+    private Connection connection;
```

Comment:
1. move this connection configuration in separate `package config ConnectionConfig` class
2.  into this class create connection in method, for example:
```java
    public static Connection getConnection() throws SQLException {
        return DriverManager.getConnection(jdbcUrl, username, password );
    }
```
3. extract url, username and password into separate text file, put it into resources and read values from there